src/bin/pg_dump/dump_babel_utils.c

```diff
@@ -204,6 +204,16 @@ dumpBabelGUCs(Archive *fout)
 		free(oid);
 	}
 
+	/*
```

Comment:
this is not clear. How setting session_replication_role will disable triggers? what are the side effects of setting this GUC? If PG does not allow triggers outside super user, why we need it outside super user?

Reply:
As discussed offline, it is fine to use this GUC.